src/ca/ubc/cs304/database/tables/Rental.sql

```diff
@@ -1,16 +1,16 @@
 CREATE TABLE Rental(
-    rID INTEGER,
-    vID INTEGER,
-    cellNum VARCHAR(255),
-    fromDate DATE,
-    fromTime VARCHAR(255),
-    toDate DATE,
-    toTime VARCHAR(255),
-    odometer VARCHAR(255),
-    cardName VARCHAR(255),
-    cardNo VARCHAR(255),
-    expDate VARCHAR(255),
-    confNo INTEGER,
+    rID INTEGER not null,
+    vID INTEGER null,
```

Comment:
By default these are already NULL if not set, so we do not need these. Also, to set a default constraint, we need to write `DEFAULT <val>` replacing <val> with our default value.